local/path/config/path-values.yaml

```diff
@@ -0,0 +1,46 @@
+global:
```

Comment:
Why is this not part of the `helm-charts` repo?

Reply:
It IS part of the Path helm chart. 

The global directive encapsulates configurations that may spawn multiple times and across different resources. You can find references in [deployment](https://github.com/buildwithgrove/helm-charts/blob/main/charts/path/templates/deployment.yaml) and [serviceaccount](https://github.com/buildwithgrove/helm-charts/blob/main/charts/path/templates/serviceaccount.yaml) resources.